src/sidebars/favourites/tearout-directive.js

```diff
@@ -157,12 +157,11 @@
                         }
 
                         function boundsChangingEvent() {
-                            // Check if you are over a drop target by seeing if the tearout rectangle intersects the drop target
-                            insideFavouritesPane = geometryService.elementIntersect(
-                                tearoutWindow, window, document.getElementsByClassName('favourites')[0]);
-
-                            if (insideFavouritesPane) {
+                            if (dragService.overThisInstance(TEAR_IN_SELECTOR)) {
                                 reorderFavourites();
+                            } else {
+                                // Check intersections to set the tear in indicator states.
+                                dragService.overAnotherInstance(TEAR_IN_SELECTOR);
```

Comment:
I've brought out the main logic I was using `overAnotherInstance` for into its own function to do the update, and updated `overAnotherInstance` to use it (so hopefully this function would provide a foundation for other-instance checks if needed).